spec/models/transcripter_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../../lib/transcripter'
+
+describe Transcripter do
+  it 'produces HTML' do
+    tei = File.read(__dir__ + '/../fixtures/transcript/mock-transcript.xml')
+    expect(Transcripter.from_tei(tei)).to eq(File.read(__dir__ + '/../fixtures/transcript/mock-transcript.html'))
+  end
+end
```

Comment:
Can you wrap unit tests in `describe method_name`? It makes it more explicit about exactly what is being tested. E.g...

```ruby
describe Transcipter do
  describe '.from_tei' do
    it 'produces HTML' do
```